From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 24 Aug 2019 19:00:41 +0100
Subject: Documentation: Fix broken link to CIPSO draft
Forwarded: not-needed

We exclude the CIPSO draft text as its licence is not DFSG compliant.
Link to the IETF's online version instead.

---
--- a/Documentation/netlabel/draft_ietf.rst
+++ b/Documentation/netlabel/draft_ietf.rst
@@ -1,5 +1,4 @@
 Draft IETF CIPSO IP Security
 ----------------------------
 
- .. include:: draft-ietf-cipso-ipsecurity-01.txt
-    :literal:
+https://tools.ietf.org/html/draft-ietf-cipso-ipsecurity-01
